src/Assignment4/App.js

```diff
@@ -20,6 +22,16 @@ const appRouter = createBrowserRouter([
         path: "/user/:id",
         element: <SpecificUserComponent />,
       },
+      {
+        path: "/about-me",
+        element: <AboutMe />,
+        children: [
+          {
+            path: "/about-me/profile", // path here can be written as /about-me/profile or profile.
```

Comment:
what if this about-me is not same as parent. does that entirely appended?

Reply:
here in the path if instead of "/about-me/profile",if  I give something other than parent (say "search/profile" )-> it will throw run time error : 
**Error: Absolute route path "/search/profile" nested under path "/about-me" is not valid. An absolute child route path must start with the combined path of all its parent routes.**

![image](https://user-images.githubusercontent.com/30456511/207247164-144f81f0-f6a6-4b22-9eca-c5e558bc0bfc.png)